script/preCache.js

```diff
@@ -9,17 +9,21 @@ module.exports = breadboard({
     'package.json': require('../package.json')
   },
   entry: ({
+    '/config/index': getConfig,
     '/lib/bundler/index': bundle,
     '/lib/getComponentInfo': getComponentInfo
   }) => {
+    const config = getConfig();
     const components = getComponentInfo().map(componentInfo => componentInfo.name );
+    const preCacheComponentBundles = config.components && config.components.preCacheBundles ? config.components.preCacheBundles : [];
```

Comment:
The`&&` line is to check that `config.components` is there before we call `config.components.preCacheBundles`. This should be fine actually in real life just the tests were breaking if we didnt set config.components